transform/snowflake-dbt/models/mattermost/downloads.sql

```diff
@@ -4,39 +4,121 @@
   })
 }}
 
-WITH downloads AS (
+WITH outliers         AS (
```

Comment:
I almost want to call this `download_stats` instead of `outliers`

Reply:
I agree.